set.mm

```diff
@@ -666258,6 +666624,670 @@ is real and it is less than or equal to its inferior limit (in such a
       DEFGRUAUB $.
   $}
 
+$(
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+                Limits for sequences of extended real numbers
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
```

Comment:
blank line after the second line -.-.-.-.-. and similarly blank line before $)

Reply:
Change as suggested (in the latest commit). Please let me know if it's ok now, I've searched the database and I see some inconsistencies.

Is this something the "check formatting" or the reformatting script could take care of?

Reply:
Not enforced nor corrected yet, but we're trying to be a bit more consistent. Discussed in #2262. Ping @avekens if he wants to add something.

Reply:
It's work in progress - fine for me.